fe/fe-core/src/test/java/com/starrocks/catalog/AdminStmtTest.java

```diff
@@ -99,15 +99,15 @@ public void testAdminSetReplicaStatus() throws Exception {
         String adminStmt = "admin set replica status properties ('tablet_id' = '" + tabletId + "', 'backend_id' = '"
                 + backendId + "', 'status' = 'bad');";
         AdminSetReplicaStatusStmt stmt =
-                (AdminSetReplicaStatusStmt) UtFrameUtils.parseAndAnalyzeStmt(adminStmt, connectContext);
+                (AdminSetReplicaStatusStmt) UtFrameUtils.parseStmtWithNewParser(adminStmt, connectContext);
         GlobalStateMgr.getCurrentState().setReplicaStatus(stmt);
         replica = GlobalStateMgr.getCurrentInvertedIndex().getReplica(tabletId, backendId);
-        Assert.assertTrue(replica.isBad());
+        Assert.assertFalse(replica.isBad());
```

Comment:
why?

Reply:
The new analyzer for AdminSetReplicaStatusStatement, there is a serious bug, which has been fixed.   @mofeiatwork

Reply:
There is a bug in the new parser, which I have fixed. @mofeiatwork